<!--/image_raw/compressed :: /camera_right_hand/image_raw/compressed,    /kinect2/qhd/image_color    /segmentation  -->
<launch>

  <node pkg="terrain_feature_fuser" type="terrain_feature_fuser" name="terrain_feature_fuser" output="screen">
    <param name="image_transport" value="raw" />
    <remap from="/points_raw" to="/cloud_filtered" />

    <remap from="/image_raw" to="/kinect2/qhd/image_color" />
    <remap from="/image_seg" to="segmentation" />
  </node> 
  
</launch>
